tests/integration/framework/binary/binary.go

```diff
@@ -60,7 +60,13 @@ func Build(t *testing.T, name string) {
 
 		// Use a consistent temp dir for the binary so that the binary is cached on
 		// subsequent runs.
-		binPath := filepath.Join(os.TempDir(), "dapr_integration_tests/"+name)
+		var tmpdir string
+		if runtime.GOOS == "darwin" {
+			tmpdir = "/tmp"
+		} else {
+			tmpdir = os.TempDir()
```

Comment:
Good catch!

Reply:
Is it a design issue for Go os package? Maybe we can submit an issue.

Reply:
I actually think Go is doing the more correct thing here as a sane default, however because we want a slightly more persistent target directory we need to divert from the default.